### DISCUSSION TITLE
ContentContainer的高度不应该是全屏match_parent,应该是屏幕高度减去键盘高度

### DISCUSSION THREAD

**Discussion by zkoalas** [DSAppTeam/PanelSwitchHelper#164] [General]
当键盘弹出时我发现,RelativeContentContainer,...的高度有问题,不应该是全屏,应该是:
ContentContainer的高度不应该是全屏match_parent,应该是屏幕高度减去键盘高度

**Comment by YummyLau** [comment]
这个高度在框架内部处理了，可以看PanelSwitchLayout的实现。
